src/shared/components/Paginator.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import type { LinkProps } from "react-router-dom";
+import type { ForwardAttributes, Pagination } from "../types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import "./styles/Paginator.scss";
+
+export const PaginatorItem: React.FC<ForwardAttributes & LinkProps> = ({
+  className,
+  children,
+  ...props
+}) => {
+  return (
+    <li className={`Paginator_item ${className || ""}`}>
+      <Link {...props} className="Paginator_link">
+        {children}
+      </Link>
+    </li>
+  );
+};
+
+class Paginator<T> extends React.Component<
```

Comment:
Curious as to why this is a class component?

Reply:
Added documentation: 9a1440ba

Reply:
Made moot with 38007020